server/server.go

```diff
@@ -183,7 +202,10 @@ func main() {
 		log.Fatal(err)
 	}
 
+	telemetry.SetStaleProjectScanner(store.GetStaleProjects)
```

Comment:
I don't see the added function to the store in this PR. Why is that? was there another PR adding that?

Reply:
Yeah this was in a separate PR to add the database functions for stale projects.